### DISCUSSION TITLE
JSON CODE Window Covering for sonoff touch 2gang

### DISCUSSION THREAD

**Discussion by jazevedodesigner** [RavenSystem/esp-homekit-devices#1786] [MEPLHAA Scripts]
Hello,  anyone have the JSON CODE for window covering? For the model sonoff touch 2 gang? I'm putting this one, 

{
  "c": { "l": 0, "b": [{ "g": 2, "p": 0, "t": 5 }], "0": { "r": [{ "g": 15 }, { "g": 16 }] } },
  "a": [{
    "t": 45,
    "o": 18,
    "c": 14,
    "f": 70,
    "0": { "r": [{ "g": 15, "v": 1 }, { "g": 4 }] },
    "1": { "r": [{ "g": 15 }, { "g": 4, "v": 1 }] },
    "2": { "r": [{ "g": 15 }, { "g": 4 }] },
    "3": { "r": [{ "g": 15, "i": 0.2 }, { "g": 4 }] },
    "4": { "r": [{ "g": 15 }, { "g": 4, "i": 0.2 }] },
    "f0": [{ "g": 5, "i": 1, "p": 0, "t":0 }],
    "f1": [{ "g": 13, "i": 1, "p": 0, "t":0 }],
    "f2": [{ "g": 5 }, { "g": 13 }]
  }]
}

..but it doesn't work!!!!

someone can help me?

**Comment by nickforev** [comment]
Hi.
Tiger2 window covering
{"c":{"l":13,"i":0,"n":"Tiger2-WC","b":[{"g":0,"t":5}],"f":13},"a":[{"t":45,"o":18,"c":14,"f":70,"0":{"r":[{"g":5,"v":1},{"g":12}]},"1":{"r":[{"g":5},{"g":12,"v":1}]},"2":{"r":[{"g":5},{"g":12}]},"3":{"r":[{"g":5,"i":0.2},{"g":12}]},"4":{"r":[{"g":5},{"g":5,"i":0.2}]},"f3":[{"g":9,"t":0}],"f4":[{"g":0,"t":0}]}]}
